deploy/firesim

```diff
@@ -96,22 +96,12 @@ def register_task(task: Callable) -> Callable:
 
 
 @register_task
-def managerinit() -> None:
-    """ Setup FireSim manager components. """
-
-    valid_creds = valid_aws_configure_creds()
-    while not valid_creds:
-        # only run aws configure if we cannot already find valid creds
-        # this loops calling valid_aws_configure_creds until
-        rootLogger.info("Running aws configure. You must specify your AWS account info here to use the FireSim Manager.")
-        # DO NOT wrap this local call with StreamLogger, we don't want creds to get
-        # stored in the log
-        local("aws configure")
-
-        # check again
-        valid_creds = valid_aws_configure_creds()
-        if not valid_creds:
-            rootLogger.info("Invalid AWS credentials. Try again.")
+def managerinit(args: argparse.Namespace):
+    """ Setup local FireSim manager components. """
+
+    if not args.platform:
+        rootLogger.error('::ERROR:: managerinit requires that you provide --platform <platformname>')
```

Comment:
Note for later, we should unify how we add "ERROR", "WARNING" to strings

Reply:
I agree.  And it probably should be done with the logging formatter(s) instead of at each caller.

Reply:
https://github.com/firesim/firesim/issues/1047